### PR TITLE
fix observation from ray

### PR DESCRIPTION
**Pull request by noskill**

### CODE REVIEW

File: src/main/java/io/singularitynet/MissionHandlers/ObservationFromRayImplementation.java [hunk 1]
```diff
@@ -57,7 +56,8 @@ public void cleanup()
     public static void buildMouseOverData(JsonObject json, boolean includeNBTData)
     {
         MinecraftClient client = MinecraftClient.getInstance();
-        HitResult mop = client.crosshairTarget;
+        Entity cameraEntity = client.getCameraEntity();
+        HitResult mop = cameraEntity.raycast(96.0, 1.0F, true);
```
Comment (CICS-Oleg) [inline src/main/java/io/singularitynet/MissionHandlers/ObservationFromRayImplementation.java:60]: Why is maxDistance 96.0?

Comment (CICS-Oleg) [reply src/main/java/io/singularitynet/MissionHandlers/ObservationFromRayImplementation.java:60]: Do we have getReachDistance() in our mod?

Comment (CICS-Oleg) [reply src/main/java/io/singularitynet/MissionHandlers/ObservationFromRayImplementation.java:60]: Yeah, there is variable `double reachDistance = 50.0f; ` we can change it if needed.

**Review by DaddyWesker — APPROVED** [review]

**Review by Necr0x0Der — APPROVED** [review]

File: src/main/java/io/singularitynet/MissionHandlers/ObservationFromRayImplementation.java (hunk 1 above)
Comment (noskill) [reply src/main/java/io/singularitynet/MissionHandlers/ObservationFromRayImplementation.java:60]: @CICS-Oleg default reach distance is 4 in minecraft. That what was used before this PR in observationFromRay

**Pull request merged**